index.html

```diff
@@ -75,5 +73,277 @@ <h5>LET'S CONNECT</h5>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+
+              <p class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="see-link">
+                <a href="#">See Project</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="mx-container">
+      <div class="hero">
+        <div>
+          <h2 class="hero-name">About</h2>
+        </div>
+        <div>
+          <h2 class="hero-msg">Myself</h2>
+        </div>
+        <div class="hero-minor-text">
+          <p>
+            I'm a software! I can help you build a product, feature or website
+            Look through some of my work and experience! If you like what you
+            see and have a project you need coded, don't hestiate to contact me.
+          </p>
+        </div>
+      </div>
+      <div>
+        <h5 class="text-connect">LET'S CONNECT</h5>
+        <div>
+          <ul class="icons-hero">
+            <li>
+              <img src="./img/twitter.svg" alt="twitter logo" />
+            </li>
+            <li>
+              <img src="./img/linkedin.svg" alt="linkedin logo" />
+            </li>
+            <li>
+              <img src="./img/medium.svg" alt="medium logo" />
+            </li>
+            <li>
+              <img src="./img/github.svg" alt="github logo" />
+            </li>
+            <li>
+              <img src="./img/angellist.svg" alt="angellist logo" />
+            </li>
+          </ul>
+        </div>
+        <button class="see-link">
+          <a href="#">Get my resume</a>
+        </button>
+      </div>
+    </section>
+    <aside>
+      <div class="lang-header">
+        <div>
+          <h5>Languages</h5>
+        </div>
+        <div>
+          <img src="./img/Union.svg" alt="down arrow" />
+        </div>
+      </div>
+      <ul class="lang-container">
+        <li class="lang">
+          <div>
+            <img src="./img/lang1.svg" alt="javascript" />
+          </div>
+          <div>
+            <h6 class="lang-text">JavaScript</h6>
+          </div>
+        </li>
+        <li class="lang">
+          <div>
+            <img src="./img/lang2.svg" alt="html" />
+          </div>
+          <div>
+            <h6 class="lang-text">HTML</h6>
```

Comment:
- [ ] Please note that this should not be a heading but just a text to show the language you are good at. Kindly change it from a heading to a paragraph.

Reply:
:heavy_check_mark: